ansible/roles/schulcloud-server-tspsync/templates/api-tsp-sync-svc.yml.j2

```diff
@@ -5,9 +5,6 @@ metadata:
   namespace: {{ NAMESPACE }}
   labels:
     app: api-tsp-sync
-  annotations:
-    prometheus.io/scrape: 'true'
-    prometheus.io/port: '3030'
```

Comment:
It's fine, but please make you an follow up ticket where you add the new metric endpoint to this service.

Reply:
Can you please create the ticket? I don't know what needs to be done. The comment of @bn-pass sounded to me as if actually nothing needs to be added here. You should clarify that.

Reply:
it's done https://ticketsystem.dbildungscloud.de/browse/BC-4707